src/Configuration/Accounts.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace ClassicUO.Configuration
+{
+    internal class Account 
```

Comment:
Please make this class immutable, or at least provide a `ctor` which initializes the properties already.

Reply:
yeah I'm not sure it can be 100% immutable.  Depends on the JSON Serializer.  Ill try it out and see, but i think the setters need to remain public. I'll add a constructor though for sure.

Yeah, as i suspected that is a no go. I will add an additional contstructor though.

Reply:
Oki, I wonder why we have to use such a limited serializer tho ;)

Reply:
yeah I'm pretty sure newtonsoft supports private setters.  I just checked the TinyJson repo and it doesn't appear to be possible.